olp-cpp-sdk-core/include/olp/core/http/NetworkResponse.h

```diff
@@ -32,19 +32,6 @@ namespace http {
  */
 class CORE_API NetworkResponse final {
  public:
-  /**
-   * @brief Check if associated request was cancelled.
-   * @return true if associated request was cancelled.
-   */
-  bool IsCancelled() const;
```

Comment:
Might be helpfull to keep this and instead of using the bool use the status code, e.g. 

```cpp
bool IsCancelled() const {
  return status_ == ErrorCode::CANCELLED_ERROR;
}
```

Reply:
Added a check